scripts/cnn_variant_wdl/cram2filtered.wdl

```diff
@@ -0,0 +1,473 @@
+# This workflow takes an input CRAM to call variants with HaplotypeCaller and filter the calls with a neural net
```

Comment:
Does it work on BAM?  If not, then it might be worth noting that this would be run after the single-sample pipeline (https://github.com/gatk-workflows/broad-prod-wgs-germline-snps-indels/blob/master/PairedEndSingleSampleWf.wdl) has already produced a CRAM.

Reply:
Currently, it is CRAM only, comment added.